infrastructure/terraform/environments/staging-eu-west-2/staging/vars

```diff
@@ -0,0 +1,26 @@
+aws_bucket = "staging-eu-west-2.terraform-state.ch.gov.uk"
+remote_state_bucket = "ch-service-staging-terraform-state"
+aws_region = "eu-west-2"
+environment = "staging"
+deploy_to = "staging"
+state_prefix = "env:/staging"
+workspace_key_prefix = "test-data-generator-stack"
+state_file_name = "test-data-generator.tfstate"
+
+# Allows internal access to servers from these addresses.
+internal_cidrs = "172.16.0.0/16,172.19.0.0/16,172.24.0.0/16,10.172.16.0/24,10.172.21.0/24"
+
+aws_profile = "staging-eu-west-2"
+ec2_instance_type = "t3.medium"
+
+# Certificate for https access through ALB
+ssl_certificate_id = "arn:aws:iam::250991044064:server-certificate/Preprod2020"
+# zone_id = "" no route53 in staging
```

Comment:
I wouldn't include commented out values. Add the values you need and and any further detail to the pull request description. Otherwise it gets confusing and there's the risk people will change the values and not the comments at which point it get really confusing!

Reply:
OK, removed the parameter.